src/Phinx/Console/Command/Create.php

```diff
@@ -311,5 +311,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         }
 
         $output->writeln('<info>created</info> ' . str_replace(getcwd() . DIRECTORY_SEPARATOR, '', $filePath));
+
+        return 0;
```

Comment:
We have class constants for those afaik

Reply:
When looking for a convention in this namespace, I found integers for existing `execute()` default return codes.

In the scope of the version bump, it OK for me to keep following that pattern?